<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpertDiscoveryPosNegDialog</class>
 <widget class="QDialog" name="ExpertDiscoveryPosNegDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Load positive and negative sequence bases</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/repeat_finder/repeats_image</normaloff>:/repeat_finder/repeats_image</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Positive sequences file</string>
       </property>
       <property name="buddy">
        <cstring>firstFileEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="firstFileEdit"/>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="openFirstButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="oneSequenceCheckBox">
       <property name="text">
        <string>Generate negative sequences</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Negative sequences file</string>
       </property>
       <property name="buddy">
        <cstring>secondFileEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QLineEdit" name="secondFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QToolButton" name="openSecondButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="instructionLabel">
       <property name="text">
        <string>This is the first step of creating a new ExpertDiscovery project.
Load the sequences you want to analyze by choosing any file with a sequence or multiple sequences.

Positive sequence base contains a regulation object you are interested in.
Negative sequence base doesn't have it. 
You also may generate negative sequences automatically.

ExpertDiscovery will extract complex signals which reflect a structure of your regulation object. 
The more sequences you provide the better will be the result</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Negatives per positive amount:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="negPerPositivespinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>openFirstButton</tabstop>
  <tabstop>firstFileEdit</tabstop>
  <tabstop>oneSequenceCheckBox</tabstop>
  <tabstop>openSecondButton</tabstop>
  <tabstop>secondFileEdit</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>ExpertDiscoveryPosNegDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ExpertDiscoveryPosNegDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
